content/en/security_platform/application_security/custom_rules.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Custom Detection Rules
+kind: documentation
+further_reading:
+- link: "/security_platform/application_security/"
+  tag: "Documentation"
+  text: "Monitoring threats with Datadog Application Security"
+- link: "/security_platform/application_security/troubleshooting"
+  tag: "Documentation"
+  text: "Troubleshoot common Datadog Application Security issues"
+---
+
+## Overview
+
+Application Security comes with a set of [out-of-the-box detection rules][1] which aim to catch attack attempts and vulnerability triggers that impact your production systems.
+
+However, there are situations where you may want to customize a rule based on your environment. For example, you may want to customize a rule that catches attack attempts on a service’s route. For instance, a pre-production development route accepting SQL and returning the results. Catching SQL attempts is noisy as the route is restricted to internal developers, therefore you can customize this rule to exclude these patterns.
```

Comment:
However, there are situations where you may want to customize a **detection** rule...